pkgs/tools/audio/beets/builtin-plugins.nix

```diff
@@ -10,6 +10,11 @@
   ...
 }:
 {
+  _typing = {
+    testPaths = [ ];
+    enable = false;
+    builtin = false;
```

Comment:
Are you sure this isn't `builtin`?

Reply:
it *is* builtin for the new version but does not exist in 2.10 (assuming I understand what builtin means)